go-app-sms.js

```diff
@@ -124,8 +124,15 @@ go.utils = {
 
     update_status: function(im, contact, status) {
         contact.extra.status = status;
+        var message_set_id = im.config.welcome_message_sets[status];
+        var payload = {
+            contact_key: contact.key,
+            messageset_id: message_set_id,
+            schedule: 1
```

Comment:
Hardcoding the schedule here is crazy, but it's what the existing code does, so we can live with it for now.